### PR TITLE
DOC-5708-WhatsNew

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-5708

Add new what's new content

### CODE REVIEW

**Comment by ibsoln** [comment]
@couchbase/documentation-writers please approve to facilitate merge to staging for full doc review

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,12 +1,17 @@
 = What's New
 
 == Highlights
+Two Java build options are available at this release:
```
Comment (eric-schneider) [inline modules/ROOT/pages/index.adoc]:
```suggestion

Two Java build options are available in this release:
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
-=== Custom Conflict Resolution
+*  Java Android - enabling development and deployment of Java apps in an android environment (Read more: xref:java-android.adoc[Java Android])
```
Comment (eric-schneider) [inline modules/ROOT/pages/index.adoc]:
```suggestion
*  Java Android - enabling development and deployment of Java apps in an Android environment (Read more: xref:java-android.adoc[Java Android])
```

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (eric-schneider) [reply modules/ROOT/pages/index.adoc]: Changed 'at' to 'in' to better conform to American English. Also added a leading line for cleaner/more consistent syntax.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+* Java Platform
+ - enabling development and deployment of Java apps in any environment that supports the JVM model.
```
Comment (eric-schneider) [inline modules/ROOT/pages/index.adoc]: Not sure if this list item is intentionally different than the one the precedes it, but it renders as a sub-bullet and doesn't look like it's formatted correctly.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
 
-The Custom Conflict Resolution feature enhances the automatic conflict resolution feature in Couchbase Mobile 2.x to allow applications to use custom logic to select the winner between conflicting revisions of a document.
+=== Java Platform
```
Comment (eric-schneider) [inline modules/ROOT/pages/index.adoc]: This section seems like it's just repeating the the content from the list item that immediately precedes it. My suggestion would be to consolidate the content into one of these places.

**Review by eric-schneider — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (ibsoln) [reply modules/ROOT/pages/index.adoc]: Consolidated as suggested

**Review by eric-schneider — APPROVED** [review]

**Pull request merged**